frontend/azlinux/handle_rpm.go

```diff
@@ -49,6 +49,10 @@ func handleRPM(w worker) gwclient.BuildFunc {
 				return nil, nil, err
 			}
 
+			if err := ref.Evaluate(ctx); err != nil {
```

Comment:
Since `runTests` now constructs the state that installs packages in the target container, this will actually unlazy the package build before the install in the target container, right?

Reply:
Correct.